app/assets/stylesheets/components/_index.scss

```diff
@@ -1 +1,2 @@
 @import "_navbar";
+@import "_footer";
```

Comment:
This is having a conflct with master, you'll need to solve that.
Also, you don't need the underscore when importing partials